<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pub</class>
 <widget class="QDialog" name="Pub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>30</y>
     <width>1181</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1171</width>
       <height>581</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Gestion publications</string>
      </attribute>
      <widget class="QTableView" name="tableViewPub">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>110</y>
         <width>661</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>401</width>
         <height>421</height>
        </rect>
       </property>
       <property name="title">
        <string>Gestion des publications</string>
       </property>
       <widget class="QComboBox" name="comboBoxTypePub">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>60</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Article</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Projet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Conference</string>
         </property>
        </item>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEditDescPub">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>301</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type publication</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Description publication</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAjouterPub">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>330</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonModifierPub">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>330</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAnuulerPub">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>360</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>485</x>
         <y>80</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditRecherchePub">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxChampPub">
       <property name="geometry">
        <rect>
         <x>702</x>
         <y>80</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Date</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Description</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBoxOrderPub">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>80</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Ascendant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Descendant</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButtonTrierPub">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonGenererPdfPub">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>470</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>GÃ©nerer PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonStatistiquePub">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>470</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Consulter Statistique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSupprimerPub">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>470</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAjouterfavoris">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>470</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter aux favoris</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>-30</x>
         <y>-20</y>
         <width>1271</width>
         <height>601</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../wallpaperflare.com_wallpaper (1) (1).png</pixmap>
       </property>
      </widget>
      <zorder>label_4</zorder>
      <zorder>tableViewPub</zorder>
      <zorder>groupBox</zorder>
      <zorder>label_3</zorder>
      <zorder>lineEditRecherchePub</zorder>
      <zorder>comboBoxChampPub</zorder>
      <zorder>comboBoxOrderPub</zorder>
      <zorder>pushButtonTrierPub</zorder>
      <zorder>pushButtonGenererPdfPub</zorder>
      <zorder>pushButtonStatistiquePub</zorder>
      <zorder>pushButtonSupprimerPub</zorder>
      <zorder>pushButtonAjouterfavoris</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Favoris</string>
      </attribute>
      <widget class="QTableView" name="tableViewPubFAvoris">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>751</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRetirerFavoris">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>460</y>
         <width>161</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Retirer des favoris</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>-100</x>
         <y>-50</y>
         <width>1361</width>
         <height>641</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../wallpaperflare.com_wallpaper (1) (1).png</pixmap>
       </property>
      </widget>
      <zorder>label_5</zorder>
      <zorder>tableViewPubFAvoris</zorder>
      <zorder>pushButtonRetirerFavoris</zorder>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Historique</string>
      </attribute>
      <widget class="QTableView" name="tableViewHistoriquePub">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>431</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>-60</x>
         <y>-20</y>
         <width>1331</width>
         <height>601</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../wallpaperflare.com_wallpaper (1) (1).png</pixmap>
       </property>
      </widget>
      <zorder>label_6</zorder>
      <zorder>tableViewHistoriquePub</zorder>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>200</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>-55</x>
         <y>-15</y>
         <width>1271</width>
         <height>591</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../wallpaperflare.com_wallpaper (1) (1).png</pixmap>
       </property>
      </widget>
      <zorder>label_7</zorder>
      <zorder>comboBox</zorder>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
